ui/packages/catalog-ui-search/src/main/webapp/component/result-selector/result-selector.view.js

```diff
@@ -25,9 +25,11 @@ const cql = require('../../js/cql.js')
 const ResultSortDropdownView = require('../dropdown/result-sort/dropdown.result-sort.view.js')
 const user = require('../singletons/user-instance.js')
 const ResultStatusView = require('../result-status/result-status.view.js')
+const store = require('../../js/store.js')
```

Comment:
I should have been more specific, but this shouldn't be necessary.  Instead of passing store, pass the selectionInterface.  If we don't do this, then some of the routes won't work.

However, a better option is to pass `this.model` down to the component instead since that's the search you need.

Reply:
Ahhh, I see what you mean. I pushed the changes and passed down `this.model` to the component and used that instead.